source-data/us-state-codes.tsv

```diff
@@ -0,0 +1,58 @@
+# from https://gist.github.com/rogerallen/1583593
```

Comment:
Some of the names here don't match the official names from the US Census. Any reason not the use the official names like we do in ID3C? https://github.com/seattleflu/id3c-customizations/blob/master/locations/data/states.txt

Reply:
ditto mangling to fit GISAID data.